tests/Microsoft.Bot.Builder.Tests/TranscriptUtilities.cs

```diff
@@ -24,19 +26,162 @@ public static class TranscriptUtilities
         /// <returns>A list of activities to test</returns>
         public static IEnumerable<IActivity> GetFromTestContext(TestContext context)
         {
-            var transcriptsRootFolder = TestUtilities.GetKey("TranscriptsRootFolder") ?? @"..\..\..\..\..\transcripts";
-            var directory = Path.Combine(transcriptsRootFolder, context.FullyQualifiedTestClassName.Split('.').Last());
-            var fileName = $"{context.TestName}.transcript";
-            var path = Path.Combine(directory, fileName);
+            var relativePath = Path.Combine(context.FullyQualifiedTestClassName.Split('.').Last(), $"{context.TestName}.chat");
+            return GetActivities(relativePath);
```

Comment:
Please add in a detailed comment with example here. This seems fragile, and as we need to debug it down the line, having a detailed comment here will be very useful.